### PR TITLE
Debugger for nimbus!

### PR DESCRIPTION
**Pull request by benrudolph**
@czue @wpride after many times of looking at this code and then just putting it off, this finally enables debugger for nimbus (and still supports old cloudcare debugger). it also clears the debugger when you leave the form. it requires this:  https://github.com/dimagi/formplayer/pull/167

the only "clean" way of doing this was to use the existing adapter and just route that call to HQ from formplayer instead of hitting HQ directly

cc: @NoahCarnahan

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js, line 431 - 'e' is defined but never used (no-unused-vars)
* corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js, line 1 - 'CodeMirror' is defined but never used (no-unused-vars)
* corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js, line 381 - Expected indentation of 8 space characters but found 12. (indent)
* corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js, line 382 - Expected indentation of 4 space characters but found 8. (indent)
* corehq/apps/cloudcare/tests/test_api.py, line 379 - W391 blank line at end of file

**Comment by wpride** [comment]
😂

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js [hunk 1]
```diff
@@ -420,12 +421,41 @@ Formplayer.ViewModels.CloudCareDebugger = function() {
...
     self.evalXPath = new Formplayer.ViewModels.EvaluateXPath();
     self.isMinimized = ko.observable(true);
+    self.instanceXml = ko.observable('');
+    self.formattedQuestionsHtml = ko.observable('');
     self.toggleState = function() {
         self.isMinimized(!self.isMinimized());
     };
 
+    $.unsubscribe('debugger.update');
+    $.subscribe('debugger.update', function(e) {
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js]: 'e' is defined but never used (no-unused-vars)

**Comment by czue** [comment]
sweet

**Pull request merged**